### PR TITLE
Add note to README about production data in dev

### PR DESCRIPTION
**Pull request by maxwofford**
This adds details about how to run with production data & keys in development. Running with the development-only setup while local is still the recommended process, but this adds documentation for currently undocumented features.

### CODE REVIEW

**Review by cjdenio — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -69,6 +71,15 @@ Visit [localhost:3000](http://localhost:3000) to see the result.
 
 **What's Solargraph?** [Solargraph](https://solargraph.org/) is a Ruby language server that provides better Intellisense and code completion. It's completely optional to use Solargraph but highly recommended. You may also need to install the [Solargraph extension](https://github.com/castwide/solargraph#using-solargraph) for your IDE.
 
+### Production Access
+
+We've transitioned to using development keys and tokens in development, but historically have used production data on dev machines. It is recommended to not roll back to using production data & keys in development, but if absolutely neccessary the following steps can be taken:
+
+- Set `USE_PROD_CREDENTIALS=true` in your docker container
+- Temporarily remove `config/credentials/development.yml.enc`
```
Comment (garyhtou) [inline README.md]: this doesn't seem to be necessary for me

File: README.md [hunk 2]
```diff
@@ -69,6 +71,15 @@ Visit [localhost:3000](http://localhost:3000) to see the result.
...
+
```
Comment (garyhtou) [inline README.md]: Put production key in config/master.key

File: README.md [hunk 3]
```diff
@@ -69,6 +71,15 @@ Visit [localhost:3000](http://localhost:3000) to see the result.
 
 **What's Solargraph?** [Solargraph](https://solargraph.org/) is a Ruby language server that provides better Intellisense and code completion. It's completely optional to use Solargraph but highly recommended. You may also need to install the [Solargraph extension](https://github.com/castwide/solargraph#using-solargraph) for your IDE.
 
+### Production Access
+
+We've transitioned to using development keys and tokens in development, but historically have used production data on dev machines. It is recommended to not roll back to using production data & keys in development, but if absolutely neccessary the following steps can be taken:
+
+- Set `USE_PROD_CREDENTIALS=true` in your docker container
```
Comment (garyhtou) [inline README.md]:
```suggestion
- Set environment variable `USE_PROD_CREDENTIALS=true` in your docker container
```

File: README.md (hunk 1 above)
Comment (maxwofford) [reply README.md]: This is required for `rails c`, but not for `rails s`

**Pull request merged**